docs/ref/files/uploads.txt

```diff
@@ -99,6 +99,11 @@ Subclasses of ``UploadedFile`` include:
     A file uploaded into memory (i.e. stream-to-memory). This class is used
     by the :class:`~django.core.files.uploadhandler.MemoryFileUploadHandler`.
 
+.. class:: SimpleUploadedFile
+
+    A simple representation of a file, which just has content, size, and a name.
```

Comment:
We try to avoid words such as just / simple. Although given that's the class name I'm not sure how much we can do here. 

https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#writing-style

Reply:
Thanks. I removed this documentation change from the PR. After @apollo13's suggestion there's no more reference to `SimpleUploadedFile` with my change.

For more context, the exact copy of the documentation that I used

>  A simple representation of a file, which just has content, size, and a name.

comes directly from the already existing [SimpleUploadedFile](https://github.com/django/django/blob/main/django/core/files/uploadedfile.py#L102)'s docstring. I don't think it needs to be changed. But if you or someone wants to suggest different copy I'm glad to change it in this PR.